### DISCUSSION TITLE
Configuration my neje master 2s

### DISCUSSION THREAD

**Discussion by IuriRod93** [arkypita/LaserGRBL#2069] [Q&A]
I have a neje master 2s laser and my laser is on direct. Can anyone guide me on how I can make the laser turn off and only turn on when executing the command in g-code?

**Comment by jomo487** [comment]
Does it have a uno with cnc shield? if so, the laser pwm input may have to be connected to Z max endstops not spindle as you would expect (Some kind of rev change on the shield) . laser grbl should take care of the on and off . be sure to set laser mode $32 to 1 IN EEPROM
